docs/examples/frameworks/jax/jax-basic_example.ipynb

```diff
@@ -0,0 +1,277 @@
+{
```

Comment:
<p>This reads weird for me wrt to our nomenclature. It is usually a pipeline definition function, defining the preprocessing, marked with the decorator etc. After reading "function that will be later used to create instances od DALI pipelines" I thought we are writing some helper for the purpose of JAX. Maybe mention the getting started or similar tutorial that covers the details of writing DALI pipelines?</p><p>You can also list steps of the pipeline in points, reading, decoding etc</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/4937/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done. I added <code>pipeline definition</code>	 and more detailed description of what is inside. I moved the links to pipeline tutorials here. </p><p>I left the information that this function creates and instance of the pipeline. I feel that this is necessary to distinguish it from JAX APIs that are generally functional and adding some decorator on a function gives you another functions that you can chain together.</p>